src/common/k8s-api/endpoints/helm-charts.api/get-readme.injectable.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { urlBuilderFor } from "../../../utils/buildUrl";
+import { apiBaseInjectionToken } from "../../api-base";
+
+const requestReadmeEndpoint = urlBuilderFor("/v2/charts/:repo/:name/readme");
+
+export type RequestHelmChartReadme = (repo: string, name: string, version?: string) => Promise<string>;
+
+const requestHelmChartReadmeInjectable = getInjectable({
+  id: "request-helm-chart-readme",
+  instantiate: (di): RequestHelmChartReadme => {
+    const apiBase = di.inject(apiBaseInjectionToken);
+
+    return (repo, name, version) => (
+      apiBase.get(requestReadmeEndpoint.compile({ name, repo }, { version }))
+    );
+  },
+});
+
+export default requestHelmChartReadmeInjectable;
```

Comment:
Note: this injectable is used only in renderer, yet it is in common. Stuff like this will probably organize more nicely later with that one ADR for feature-directories.

Reply:
Unless what we really wanted was to export these to the extension API. I think that they should be in common because they are implemented in a way that anyone could use them.